main.go

```diff
@@ -58,12 +59,20 @@ func main() {
 	}
 
 	var cmds []*exec.Cmd
+	// check for a .git directory and whether it's empty
+  if isDirEmpty(filepath.Join(c.Dir, ".git")) {
+		cmds = append(cmds, initGit())
+		cmds = append(cmds, remote(c))
+	}
+
+	cmds = append(cmds, fetch(c))
+
 	if isPR(c) {
-		cmds = append(cmds, clone(c))
-		cmds = append(cmds, fetch(c))
+		//cmds = append(cmds, clone(c))
+		//cmds = append(cmds, fetch(c))
```

Comment:
> FETCH_HEAD is a short-lived ref, to keep track of what has just been fetched from the remote repository

the `fetch` function explicitly fetches the pull request (c.Clone.Ref, which looks something like `refs/pull/10`, for example). Since we are no longer doing this I wonder if FETCH_HEAD will checkout the correct pull request.